docker-compose.yml

```diff
@@ -150,6 +150,78 @@ services:
       - traefik.docker.network=cryptlex-on-premise_backend
       - traefik.port=4200
 
+  admin-portal:
+    image: cryptlex/cryptlex-admin-portal:latest
+    depends_on:
+      - web-api
+    env_file: web-portals.env
+    environment:
+      WEB_API_BASE_URL: https://${WEB_API_DOMAIN}
+      RELEASE_SERVER_BASE_URL: https://${RELEASE_SERVER_DOMAIN}
+      GOOGLE_CLIENT_ID: ${GOOGLE_CLIENT_ID}
```

Comment:
@saalim-mushtaq remove google client ids from new portals and env file too as we don't support it in new portals

Reply:
@Okay